website/src/routes/routes.ts

```diff
@@ -67,3 +65,11 @@ export const routes = {
 function withOrganism(organism: string, path: `/${string}`) {
     return `/${organism}${path}`;
 }
+
+function sequenceDownloadUrl(accessionVersion: AccessionVersion | string, fileType: FileType, download = false) {
```

Comment:
nit: In nomenclature terms as a reader I would except this function to only be about downloading _sequences_. Whereas the TSV is actually no sequence, just metadata. I think `sequenceEntryDownloadUrl` would be one option.

Reply:
thanks! I like accurate naming. fixed: https://github.com/loculus-project/loculus/pull/3003/commits/7f34c6fa04d599ccd0845b7c95880e2aced0074c